### PR TITLE
Front end tests fix

### PR DESCRIPTION
**Pull request by neddoz**
Completed the missing tests regarding fron-tend on healthchecks

### CODE REVIEW

File: hc/front/tests/test_add_channel.py [hunk 1]
```diff
@@ -38,4 +38,21 @@ def test_instructions_work(self):
...
+        self.assertNotEqual(response.status_code, 403)
+
     ### Test that bad kinds don't work
+    def test_bad_kinds(self):
+        self.client.login(username='bob@example.org', password="password")
+        """ provide non existent kinds"""
+        kinds = ("kdenno", "boo", "true", "else", "yes", "no")
+        for frag in kinds:
+            url = "/integrations/add_%s/" % frag
+            r = self.client.get(url)
```
Comment (neddoz) [inline hc/front/tests/test_add_channel.py]: make variable r more descriptive

File: hc/front/tests/test_add_channel.py [hunk 2]
```diff
@@ -38,4 +38,21 @@ def test_instructions_work(self):
...
+        url = "/accounts/switch_team/alice"
+        response = self.client.get(url)
+        self.assertNotEqual(response.status_code, 403)
+
     ### Test that bad kinds don't work
+    def test_bad_kinds(self):
+        self.client.login(username='bob@example.org', password="password")
+        """ provide non existent kinds"""
+        kinds = ("kdenno", "boo", "true", "else", "yes", "no")
+        for frag in kinds:
```
Comment (neddoz) [inline hc/front/tests/test_add_channel.py]: What is frag? Make it more descriptive :)

File: hc/front/tests/test_add_channel.py [hunk 3]
```diff
@@ -38,4 +38,21 @@ def test_instructions_work(self):
...
 
     ### Test that the team access works
+    def test_team_access(self):
+        """ login as a different user"""
+        self.client.login(username='bob@example.org', password="password")
+
+        """ a url to make a request to alice's stuff using Bobs's credential who is a team member"""
+        url = "/accounts/switch_team/alice"
+        response = self.client.get(url)
+        self.assertNotEqual(response.status_code, 403)
```
Comment (njirap) [inline hc/front/tests/test_add_channel.py:48]: We need what it should be and not what it should not be

File: hc/front/tests/test_add_channel.py [hunk 4]
```diff
@@ -38,4 +38,21 @@ def test_instructions_work(self):
...
+
     ### Test that bad kinds don't work
+    def test_bad_kinds(self):
+        self.client.login(username='bob@example.org', password="password")
+        """ provide non existent kinds"""
```
Comment (njirap) [inline hc/front/tests/test_add_channel.py]: the triple quotes is put at the beginning of a class .. Single line comments are with hash

File: hc/front/tests/test_add_channel.py [hunk 5]
```diff
@@ -38,4 +38,21 @@ def test_instructions_work(self):
...
+        kinds = ("kdenno", "boo", "true", "else", "yes", "no")
```
Comment (njirap) [inline hc/front/tests/test_add_channel.py]: better descriptive variable names please

File: hc/front/tests/test_add_channel.py [hunk 6]
```diff
@@ -38,4 +38,21 @@ def test_instructions_work(self):
...
+        for kind in kinds:
```
Comment (njirap) [inline hc/front/tests/test_add_channel.py]: testing for one non-supported integration is sufficient

File: hc/front/tests/test_add_check.py [hunk 7]
```diff
@@ -12,3 +12,10 @@ def test_it_works(self):
         assert Check.objects.count() == 1
 
     ### Test that team access works
+    def test_check_team_works(self):
+        url = "/checks/add/"
+        self.client.login(username="bob@example.org", password="password")
+        response = self.client.post(url)
+
+        added_check = Check.objects.get()
```
Comment (njirap) [inline hc/front/tests/test_add_check.py]: please research on the return type of this

**Pull request merged**